src/registry.rs

```diff
@@ -268,6 +268,12 @@ lazy_static! {
     };
 }
 
+/// Default registry (global static).
+pub fn default_registry() -> &'static Registry {
+    lazy_static::initialize(&DEFAULT_REGISTRY);
```

Comment:
Or can we just deref the static created by lazy static?

Reply:
We just need to somehow ensure the Registry is initialized before handing it out, so yes we can.
That would a be a
```rust
&*DEFAULT_REGISTRY;
```
which I find a little less intuitive to understand than the current form (and I'm a bit worried about dead-code optimization), but could be explained with a comment.
Can we stick to the current form (my preference) or do you want me to swap with the one above?

Reply:
Both are fine. I was used to the `&*` form :)

Reply:
Ack. I'll keep the explicit `initialize` then.